products/sourcerepo/terraform.yaml

```diff
@@ -42,6 +42,13 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         description: |
           Resource name of the repository, of the form `{{repo}}`.
           The repo name may contain slashes. eg, `name/with/slash`
+      pubsubConfigs: !ruby/object:Overrides::Terraform::PropertyOverride
+        key_diff_suppress_func: 'compareSelfLinkOrResourceName'
+        key_expander: 'expandSourceRepoRepositoryPubsubConfigsTopic'
+        key_description: |
+          A Cloud Pub/Sub topic in this repo's project. Values are of the form
+          `projects/<project>/topics/<topic>` or `<topic>` (where the topic will
+          be inferred).
```

Comment:
Hi @rileykarson, was this meant to say "`<topic>` (where the project will be inferred)" ?

Reply:
Ah, yep.

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/pull/3368